resources/views/meetings/meeting.blade.php

```diff
@@ -22,11 +22,24 @@ class="w-full sm:max-w-md md:max-w-lg lg:max-w-xl xl:max-w-5xl mt-6 px-6 py-4 bg
             Created at: {{ $meeting->created_at }}<br>
             Updated at: {{ $meeting->updated_at }}<br>
 
-            <div class="text-right pr-16">
-                <button class="btn btn-outline border-none text-black text-xl shadow-black shadow-2xl hover:shadow-none"
+            @php
+                $hasVoted = session()->has('voted_' . $meeting->id);
+            @endphp
+
+            @if (!$hasVoted)
+                <div class="text-right pr-16">
+                    <button
+                        class="btn btn-outline border-none text-black text-xl shadow-black shadow-2xl hover:shadow-none"
                         onclick="votesModal.showModal()">Vote on times
-                </button>
-            </div>
+                    </button>
+                </div>
+            @else
+                <div class="text-right pr-16">
+                    <button class="btn btn-outline border-none text-black text-xl shadow-2xl" disabled="disabled">
+                        You have already voted
```

Comment:
The button's color will definitely have to be changed. The text is barely readable as of now